gensim/models/fasttext.py

```diff
@@ -1338,63 +1144,21 @@ def get_vector(self, word, norm=False):
             else:
                 return word_vec
 
-    def init_ngrams_weights(self, seed):
-        """Initialize the vocabulary and ngrams weights prior to training.
+    def resize_vectors(self, seed=0):
+        """Make underlying vectors match 'index_to_key' size; random-initialize any new rows.
 
-        Creates the weight matrices and initializes them with uniform random values.
+        Unlike in superclass, the 'vectors_vocab' array is of primary importance, with
+        'vectors' derived from it. And, the ngrams_vectors may need allocation."""
```

Comment:
This is particularly unhelpful. What does "may need allocation" mean here? It is too ambiguous. Does it mean the user should allocate it? Again, what are we trying to convey here?

Reply:
Yes, this looks like slop from other spliced-together code comments during the re-org.